### PR TITLE
Config file for the forwarder

### PR DESCRIPTION
**Pull request by andy-maier**

### CODE REVIEW

**Review by EdGue42 — APPROVED** [review]

File: examples/config_example.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Config file for Z HMC OS Message Forwarder
+
+hmc:
+  hmc: 10.11.12.13
```
Comment (simon-spinner) [inline examples/config_example.yaml:4]: Minor: hmc.hmc looks strange. Why not use host as we do for the zhmccli.

File: examples/config_example.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  userid: "myuser"
+  password: "mypassword"
```
Comment (simon-spinner) [inline examples/config_example.yaml:6]: Will there be a way to pass in the password via command line argument or environment variable?

File: zhmc_os_forwarder/schemas/config_schema.yaml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+          type: array
+          default: []
+          items:
+            description: "A syslog server"
+            type: object
+            required:
+              - server
+            additionalProperties: false
+            properties:
+              server:
```
Comment (simon-spinner) [inline zhmc_os_forwarder/schemas/config_schema.yaml:56]: Should we allow to specify the port? Not sure whether all syslog servers are running on the default ports. Do we assume UDP or TCP as transport layer?

File: examples/config_example.yaml (hunk 2 above)
Comment (andy-maier) [reply examples/config_example.yaml:6]: We need to do something about passwords in config files. In the most advanced form that would a user-extensible plug interface for a vault file, with a default implementation provided by us for a local vault file.
Anyone is welcome to come up with a concept like that. It should be in the zhmcclient library so we can reuse it in all packages on top of it.
But I don't want to start this now and here.

File: zhmc_os_forwarder/schemas/config_schema.yaml (hunk 3 above)
Comment (andy-maier) [reply zhmc_os_forwarder/schemas/config_schema.yaml:56]: We can add that as we learn about syslog forwarding. That's why I made the structure extensible.

**Pull request merged**